### DISCUSSION TITLE
Cache Response that lives after an page reload

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [TanStack/query#6115] [Q&A]
Hi, I want to cache my response even if the page reloads. 

With `stale time` I got so far that if my query parameters don't change it does not refetch and is instead using the cache.

 But it seems that the `stale time` cache does not live after a reload.

Is it able to store the response e.g. in the local storage, so it also does not refetch after a page reload, with tooling from tanstack/react-query?

**Comment by TkDodo** [comment]
have a look at persisters:

https://tanstack.com/query/v4/docs/react/plugins/persistQueryClient
